server/app/views/admin/programs/ProgramBlockPredicatesEditView.java

```diff
@@ -310,22 +314,27 @@ private ContainerTag createScalarDropdown(QuestionDefinition questionDefinition)
         .setLabelText("Field")
         .setCustomOptions(options)
         .addReferenceClass(ReferenceClasses.PREDICATE_SCALAR_SELECT)
-        .getContainer();
+        .getSelectTag();
   }
 
-  private ContainerTag createOperatorDropdown() {
-    ImmutableList<ContainerTag> operatorOptions =
+  private DivTag createOperatorDropdown() {
+    ImmutableList<OptionTag> operatorOptions =
         Arrays.stream(Operator.values())
             .map(
                 operator -> {
                   // Add this operator's allowed scalar types as data, so that we can determine
                   // whether to show or hide each operator based on the current type of scalar
                   // selected.
-                  ContainerTag option =
-                      option(operator.toDisplayString()).withValue(operator.name());
+                  OptionTag option = option(operator.toDisplayString()).withValue(operator.name());
                   operator
                       .getOperableTypes()
-                      .forEach(type -> option.attr("data-" + type.name().toLowerCase()));
+                      .forEach(
+                          type ->
+                              option.attr(
+                                  "data-",
+                                  type.name()
+                                      .toLowerCase())); // TODO probably a bug since a value needs
+                  // to be passed?
```

Comment:
Can we add a TODO with a specific issue number?

I'm not sure I follow on what the bug is here. Mind expanding the TODO to capture it?

Reply:
I found the with `withData` method this time